pkg/config/v2/config.go

```diff
@@ -91,6 +91,10 @@ func (EntityType) ID() TypeId {
 // AutomationResource defines which resource is an AutomationType
 type AutomationResource string
 
+func (a AutomationResource) String() string {
```

Comment:
I still don't understand why we need this. Especially now that it's not a pointer receiver anymore. 
Printing an AutomationResource value should just print it as the string it is anyway?

Reply:
You're right. removed